pontoon/pretranslation/tests/test_pretranslate.py

```diff
@@ -377,33 +374,33 @@ def test_get_pretranslations_fluent_accesskeys_prefixed_label_attribute(
     pretranslated_string = serializer.serialize_entry(parser.parse_entry(expected))
 
     response = get_pretranslation(fluent_entity, google_translate_locale)
-    assert response == (pretranslated_string, gt_user)
+    assert response == (pretranslated_string, "gt")
 
 
 @patch("pontoon.pretranslation.pretranslate.get_google_translate_data")
 @pytest.mark.django_db
 def test_get_pretranslations_fluent_accesskeys_ignore_placeables(
-    gt_mock, fluent_resource, google_translate_locale, gt_user
+    gt_mock, fluent_resource, google_translate_locale
 ):
     # Ignore placeables whene generating accesskeys
     input_string = dedent(
         """
         title = Title
-            .label = { brand } string with placeables
+            .label = { $brand } string with placeables
             .accesskey = B
     """
     )
     fluent_entity = EntityFactory(resource=fluent_resource, string=input_string)
 
     gt_mock.return_value = {
         "status": True,
-        "translation": "{ brand } gt_translation",
+        "translation": "{ $0 } gt_translation",
     }
 
     expected = dedent(
         """
-        title = { brand } gt_translation
-            .label = { brand } gt_translation
+        title = {$0} gt_translation
+            .label = { $brand } gt_translation
```

Comment:
The `title` value here is hacky, but demonstrates that a Google Translate response that includes an unexpected `{$0}` does not cause a failure, just odd output. For the `.label` it's expected, and therefore replaced with the correct placeholder.